package.json

```diff
@@ -8,7 +8,10 @@
     "generate:types": "didc bind ./src/internet_identity/internet_identity.did -t ts > src/frontend/generated/internet_identity_types.d.ts",
     "generate:js": "didc bind ./src/internet_identity/internet_identity.did -t js > src/frontend/generated/internet_identity_idl.js",
     "build": "NODE_ENV=production webpack",
-    "test": "jest --verbose",
+    "test": "jest --verbose --testPathIgnorePatterns=\"src/frontend/src/test-e2e\"",
```

Comment:
This means that running `npm test` won't run the e2e tests anymore, correct? In the future would be nice to split that out of the PR (and point it out in the PR description :))

Reply:
true, sorry :-)